### DISCUSSION TITLE
Integration with other logs

### DISCUSSION THREAD

**Discussion by 2e0wkw** [magicbug/Cloudlog#1007] [General]
One of the features of CloudLog that i love is that it integrates to all the other major logs such as LoTW, eQSL, QRZ and ClubLog.

What puzzles me is why is the technique ti sync with each log so different?

LoTW - Cron Job
eSQL - manual
QRZ and ClubLog fully automatic

Could LoTW and eSQL be made fully automatic?

Eric
2E0WKW

**Comment by magicbug** [comment]
Thought I'd explain this one, there's a few reasons why things vary between the two.

- LoTW is new code, Uploads/Downloads can be done via a cronjob this is so that page load times are fast when logging keeps it in the background plus real-time isn't needed.
- eQSL is manual purely as the code is early Cloudlog, again real-time isn't the best-added load times best handled in the background this code just needs updated to run as a cronjob
- Clublog - Background via cronjob you can set the time interval but again really doesn't need to be real-time for most.
- QRZ was a test in real-time, personally, I wouldn't run it that way, adds page load and if it times out it will error, better handled in the background via cronjobs.

**Reply by 2e0wkw** [reply]
OK, agree that CronJobs are the way to go and agree with the need to keep page load times down.

Thanks
